src/domain/community/pendingMembership/PendingMemberships.tsx

```diff
@@ -94,13 +94,17 @@ export const InvitationHydrator = ({
 
   const journey = spaceData?.lookup.space;
 
+  const userId = invitation.invitation.createdBy?.id;
+
   const { data: userData } = usePendingMembershipsUserQuery({
-    variables: {
-      userId: invitation.invitation.createdBy.id,
-    },
+    variables: { userId },
+    skip: !userId,
   });
 
-  const createdBy = userData?.lookup.user;
+  let userDisplayName: string = 'This user no longer exists on the platform!';
+  if (userData && userData.lookup.user?.profile.displayName) {
+    userDisplayName = userData.lookup.user?.profile.displayName;
+  }
```

Comment:
`const userDisplayName = userData.lookup.user?.profile.displayName ?? 'This user no longer exists on the platform!';`